app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HelloRailsReact</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
```

Comment:
- [ ] I will encourage you to create a view file to reference from your javascript folder and be rendered. Something like <div id="root">

Reply:
@Olamarx  I am not getting what you wrote here 

 I will point the javascript folder it should be.
This is the javascipt folder it should be
Something like

@Olamarx  Hello brother,I am not getting you.Could you please try to tell me in a descriptive?

Reply:
For this, there should be a root file with the root id that you will use in the React index component